# >>>TODO: this needs tunning

waypoint_A: [1.45,-2.71, 2.0]
waypoint_A_1: [1.86,3.6,2.0]
waypoint_B: [-0.519,6.91, 2.0]
waypoint_B_1: [-2.52,7.36,2.0]
waypoint_C: [-0.519,6.91, 2.0]
waypoint_C_1: [1.86,3.6,2.0]
waypoint_C_2: [1.45,-2.71, 2.0]
target_pose: [0.4, -0.1, 0.25, 0.0, 0.0, 0.0]
target_pose_1: [0.4, -0.1, 0.78, 1.57, 0.0, 0.0]